docs/reference/aggregations/bucket/datehistogram-aggregation.asciidoc

```diff
@@ -175,20 +185,20 @@ a calendar interval like month or quarter will throw an exception.
 
 The accepted units for fixed intervals are:
 
-milliseconds (ms) ::
+milliseconds (`ms`) ::
```

Comment:
Having an empty def feel weird to me. Maybe move the expanded word within the def?

```
`ms`::
Milliseconds
```

Would need to do this for all the definitions tho.